addons/skin.estuary/1080i/Variables.xml

```diff
@@ -317,9 +317,10 @@
 		<value condition="ListItem.IsRecording">windows/pvr/record.png</value>
 		<value condition="ListItem.IsPlaying">overlays/watched/OverlayPlaying-List.png</value>
 		<value condition="ListItem.IsResumable">overlays/watched/resume.png</value>
+		<value condition="ListItem.IsParentFolder">DefaultFolderBack.png</value>
+		<value condition="ListItem.IsFolder">overlays/folder.png</value>
```

Comment:
hehe, sorry. "sounds fine" wasnt really meant as a code ACK, but as a concept ACK. Will deal with it. 👍